### PR TITLE
1.2.3 & 1.2.5 Video alternatives

### PR DESCRIPTION
**Pull request by WilcoFiers**
This is a complete set of rules for HTML video for 1.2.3 and 1.2.5. All rules for those are included except for SC1-2-video-description-track, which is already published here: auto-wcag.github.io/auto-wcag/rules/SC1-2-Video-description-track.html

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).
- [ ] Make sure the pull request is prefixed with `Rule:` or `Algorithm:` or `Chore:` based on work involved.

# How to Review And Approve
- Go to the “files changed” tab, there you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “approve” and click “Submit review”.

### CODE REVIEW

File: pages/design/composed-template-empty.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- x.y.z
+
+atomic_rules:
+- 
+
+authors:
+-
+---
+
+## Aggregation Definition
```
Comment (EmmaJP) [inline pages/design/composed-template-empty.md]: Apparently the word "Aggregation" may be replaced with something else that references how a set of atomic rules work together for a composed ruled. Maybe "Composition Definition" or "Composed Definition".

File: _rules/SC1-2-video-media-alternative.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+---
```
Comment (annethyme) [inline _rules/SC1-2-video-media-alternative.md]: Should atomic rules reference the guidelines if they don't map to SC? And should we then not call it SC?

**Comment by WilcoFiers** [comment]
Review please...

File: _rules/SC1-2-3-video-alternative.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
+---
+name: video with audio has an accessible alternative
```
Comment (nitedog) [inline _rules/SC1-2-3-video-alternative.md]: I think "accessible alternative" is too broad because the rule only checks for one aspects of accessibility. Maybe "video with audio has audio descriptions or transcript"?

File: _rules/SC1-2-3-video-alternative.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+rule_type: composite
+description: |
+  This rule checks video elements with audio have an alternative available as audio or as text
```
Comment (nitedog) [inline _rules/SC1-2-3-video-alternative.md]: "alternative available" -> "alternative for the video content"

File: _rules/SC1-2-5-video-audio-alternative.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
+---
+name: video with audio has an accessible alternative
```
Comment (nitedog) [inline _rules/SC1-2-5-video-audio-alternative.md]: Same as above issue on "accessible alternative". How about "video with audio has audio description"?

File: _rules/SC1-2-5-video-audio-alternative.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](SC1-2-video-audio-description)
+- [Video element description track](SC1-2-video-description-track)
+- [Video as a media alternative for text](SC1-2-video-media-alternative)
```
Comment (nitedog) [inline _rules/SC1-2-5-video-audio-alternative.md]: I'm not sure about this. The exception is not in the Success Criterion but in the definition for "Synchronized Media". That is, if the *audio* is a media alternative, then it is not part of synchronized media. Since we are only looking at video with audio, if the audio is a media alternative then it would be video-only that is not in synchronized media. So we need to check that the audio is a media alternative rather than the video, I think.

File: _rules/SC1-2-3-video-alternative.md (hunk 3 above)
Comment (ShadowBB) [reply _rules/SC1-2-3-video-alternative.md]: Great suggestion. We have implemented this.

File: _rules/SC1-2-video-media-alternative.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+Each target element has a label indicating the `video` is an alternative to text on the page.
+
+### Expectation 3
+
+The label (from expectation 2) is [visible on the page](#visible-on-the-page) and [exposed to assistive technolgies](#exposed-to-assistive-technologies)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```
[the remark this answers is not in the document]
Comment (ShadowBB) [reply _rules/SC1-2-video-media-alternative.md]: Done. Thank you.

File: _rules/SC1-2-3-video-alternative.md (hunk 4 above)
Comment (ShadowBB) [reply _rules/SC1-2-3-video-alternative.md]: Done. Thanks.

File: _rules/SC1-2-5-video-audio-alternative.md (hunk 5 above)
Comment (ShadowBB) [reply _rules/SC1-2-5-video-audio-alternative.md]: Done. Thanks!

File: _rules/SC1-2-video-audio-description.md [hunk 8]
```diff
@@ -66,10 +70,10 @@ A video element with an audio description.
 <figure id="ozplayer-1-container" class="ozplayer-container">
```
[the remark this answers is not in the document]
Comment (ShadowBB) [reply _rules/SC1-2-video-audio-description.md]: > Please do not reference individual products, services, and vendors.

Added a note to clarify we are not endorsing this specific player.

File: _rules/SC1-2-3-video-alternative.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](SC1-2-video-audio-description)
```
Comment (jkodu) [inline _rules/SC1-2-3-video-alternative.md]: These needs to be full urls

Comment (ShadowBB) [reply _rules/SC1-2-3-video-alternative.md]: fixed. Thanks.

File: _rules/SC1-2-5-video-audio-alternative.md (hunk 6 above)
Comment (ShadowBB) [reply _rules/SC1-2-5-video-audio-alternative.md]: @jkodu This is done by the way. Thanks.

File: _rules/SC1-2-video-media-alternative.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation 1
+
+All the information contained in each target element is available as text (directly or via text alternatives) that is [visible on the page](#visible-on-the-page) and [exposed to assistive technolgies](#exposed-to-assistive-technologies).
+
+### Expectation 2
+
+Each target element has a label indicating the `video` is an alternative to text on the page.
```
Comment (nitedog) [inline _rules/SC1-2-video-media-alternative.md]: in other rules we say "is labelled as" instead of "has a label", to avoid confusing with <label> elements

File: _rules/SC1-2-video-media-alternative.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+
+### Expectation 3
+
+The label (from expectation 2) is [visible on the page](#visible-on-the-page) and [exposed to assistive technolgies](#exposed-to-assistive-technologies)
```
Comment (nitedog) [inline _rules/SC1-2-video-media-alternative.md]: "from expectation 2" seems quite procedural. I think expectations 2 and 3 should be combined.

Comment (nitedog) [reply _rules/SC1-2-video-media-alternative.md]: Also, "the label" can be confused with <label> elements. Yet I can't think of another term. Maybe add a clarification note?

File: _rules/SC1-2-video-media-alternative.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element that is [visible](#visible), where the video  contains audio.
+
+### Expectation 1
+
+All the information contained in each target element is available as text (directly or via text alternatives) that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
```
Comment (nitedog) [inline _rules/SC1-2-video-media-alternative.md]: does "text alternatives" need to be specified (or defined)?

File: _rules/SC1-2-video-media-alternative.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (nitedog) [reply _rules/SC1-2-video-media-alternative.md]: please link the word "poster" as you do in other rules (also for consistency)

File: _rules/SC1-2-video-transcript.md [hunk 13]
```diff
@@ -73,11 +72,11 @@ Then he stops to scratch his bottom.</p>
...
-<video controls>
+<video controls data-rule-target>
   <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4"></source>
   <source src="../test-assets/rabbit-video/video.webm" type="video/webm"></source>
 </video>
-<a href="/test-assets/rabbit-video-transcript.html">Transcript</p>
+<a href="../test-assets/rabbit-video/transcript.html">Transcript</p>
 ```
 
 ## Failed
```
Comment (nitedog) [inline _rules/SC1-2-video-transcript.md]: should be heading level 3 (###)?

File: _rules/SC1-2-video-transcript.md [hunk 14]
```diff
@@ -101,11 +100,11 @@ Then he stops to scratch his bottom.</p>
...
-<video controls>
+<video controls data-rule-target>
   <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4"></source>
   <source src="../test-assets/rabbit-video/video.webm" type="video/webm"></source>
 </video>
-<a href="/test-assets/rabbit-video-incorrect-transcript.html">Transcript</p>
+<a href="../test-assets/rabbit-video/incorrect-transcript.html">Transcript</p>
 ```
 
 ## Inapplicable
```
Comment (nitedog) [inline _rules/SC1-2-video-transcript.md]: should be heading level 3 (###)?

File: _rules/SC1-2-video-transcript.md [hunk 15]
```diff
@@ -21,7 +21,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video contains audio.
 
 ### Expectation 1
```
Comment (nitedog) [inline _rules/SC1-2-video-transcript.md]: Wonder if expectation 1 & 2 should be combined - the "from expectation 1" cross reference seems kind of procedural, and should be avoided IMO

File: _rules/SC1-2-video-transcript.md [hunk 16]
```diff
@@ -43,12 +43,11 @@ There are no major accessibility support issues known for this rule.
...
-- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G78
-- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G173
-- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G203
+- [Understanding 1.2.3: Audio Description or Media Alternative (Prerecorded)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc.html)
+- [Understanding 1.2.8: Media Alternative (Prerecorded)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-text-doc.html)
+- [G78: Providing a second, user-selectable, audio track that includes audio descriptions](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G78)
+- [G173: Providing a version of a movie with audio descriptions](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G173)
+- [G203: Using a static text alternative to describe a talking head video](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G203)
 
 ## Test Cases
```
Comment (nitedog) [inline _rules/SC1-2-video-transcript.md]: should be heading level 3 (###)?

File: _rules/SC1-2-video-transcript.md (hunk 15 above)
Comment (ShadowBB) [reply _rules/SC1-2-video-transcript.md]: I think my solution works. Thanks for the suggestion!

File: _rules/SC1-2-video-media-alternative.md (hunk 12 above)
Comment (ShadowBB) [reply _rules/SC1-2-video-media-alternative.md]: Now that I think about it mentioning text alternatives doesn't really add anything. We can simply remove "(directly or via text alternatives)" and it would work just fine I think. Do you agree?

File: _rules/SC1-2-3-video-alternative.md [hunk 17]
```diff
@@ -0,0 +1,183 @@
...
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-audio-description.html)
+- [video element transcript](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-transcript.html)
+- [Video element description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-description-track.html)
+- [Video as a media alternative for text](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-media-alternative.html)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```
Comment (carlosapaduarte) [inline _rules/SC1-2-3-video-alternative.md]: link poster to its definition (in the same way it's done in the video-only rules)
do this also for the atomic rules

File: _rules/SC1-2-3-video-alternative.md [hunk 18]
```diff
@@ -0,0 +1,183 @@
...
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-audio-description.html)
+- [video element transcript](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-transcript.html)
+- [Video element description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-description-track.html)
+- [Video as a media alternative for text](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-media-alternative.html)
```
Comment (carlosapaduarte) [inline _rules/SC1-2-3-video-alternative.md]: Start every rule with or without a capital letter consistently

File: _rules/SC1-2-5-video-audio-alternative.md [hunk 19]
```diff
@@ -0,0 +1,151 @@
...
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-audio-description.html)
+- [Video element description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-description-track.html)
+- [Video as a media alternative for text](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-media-alternative.html)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```
Comment (carlosapaduarte) [inline _rules/SC1-2-5-video-audio-alternative.md]: link poster to its definition (in the same way it's done in the video-only rules)

File: _rules/SC1-2-5-video-audio-alternative.md [hunk 20]
```diff
@@ -0,0 +1,151 @@
...
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-audio-description.html)
+- [Video element description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-description-track.html)
+- [Video as a media alternative for text](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-video-media-alternative.html)
```
Comment (carlosapaduarte) [inline _rules/SC1-2-5-video-audio-alternative.md]: Start every rule with or without a capital letter consistently

File: _rules/SC1-2-Video-description-track.md [hunk 21]
```diff
@@ -38,51 +38,51 @@ This means that the rule can only provide a pass for these succescriteria if acc
 
 ## Background
 
-- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc.html
-- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc-only.html
-- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G78
-- https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/H96
+- [Understanding Success Criterion 1.2.3: Audio Description or Media Alternative (Prerecorded)](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc.html)
```
Comment (carlosapaduarte) [inline _rules/SC1-2-Video-description-track.md]: (cannot comment on the Accessibility support section, but that is what is addressed in this comment)
Please review and correct several misspelling:
discription -> description
succescriteria -> success criteria
accessistive -> assistive

File: _rules/SC1-2-video-audio-description.md [hunk 22]
```diff
@@ -115,11 +117,11 @@ A video element with an incorrect audio description.
 
 <figure id="ozplayer-1-container" class="ozplayer-container">
   <div data-controls="stack" class="ozplayer" id="ozplayer-1">
-    <video controls="controls" preload="none">
+    <video data-rule-target controls="controls" preload="none">
```
Comment (carlosapaduarte) [inline _rules/SC1-2-video-audio-description.md]: Are we still using the data-rule-target ?
Should we repeat the note about the non-endorsement of the ozplayer here?

File: _rules/SC1-2-video-media-alternative.md [hunk 23]
```diff
@@ -0,0 +1,128 @@
...
+- [Understanding Success Criterion 1.2.5: Audio Description (Prerecorded)](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-audio-desc-only.html)
+- [G78:Providing a second, user-selectable, audio track that includes audio descriptions](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G78)
+- [H96:Using the track element to provide audio descriptions](https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/H96)
+
+## Test cases
+
+### Passed
+
+```html
+<!-- A video element that describes some of the text on the same page. The text on the page labels the video as an alternative.-->
```
Comment (carlosapaduarte) [inline _rules/SC1-2-video-media-alternative.md]: Move the comment outside the code as in every other rule

File: _rules/SC1-2-video-audio-description.md (hunk 22 above)
Comment (ShadowBB) [reply _rules/SC1-2-video-audio-description.md]: Can you eleborate on the "data-rule-target" question? I did not create this implementation and am unsure about answering your question.

I repeated the note. Thank you.

Comment (carlosapaduarte) [reply _rules/SC1-2-video-audio-description.md]: I was under the impression that we have stopped using the data-rule-target attribute to mark the element that was the target of the ACT rule.

File: _rules/SC1-2-3-video-alternative.md [hunk 24]
```diff
@@ -0,0 +1,183 @@
...
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
```
Comment (nitedog) [inline _rules/SC1-2-3-video-alternative.md]: (question) applies to *any* or applies to *every* occurrence? AFAIK, "any" could be any subset of occurrences on a page.

Comment (nitedog) [reply _rules/SC1-2-3-video-alternative.md]: (editorial) grammar seems incorrect - "this rule applies to any visible non-streaming video element" or "this rule applies to any non-streaming video element that is visible" or such.

File: _rules/SC1-2-5-video-audio-alternative.md [hunk 25]
```diff
@@ -0,0 +1,151 @@
...
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
```
Comment (nitedog) [inline _rules/SC1-2-5-video-audio-alternative.md]: same as above - question on any vs every and grammar issue

File: _rules/SC1-2-Video-description-track.md [hunk 26]
```diff
@@ -18,7 +19,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio and a `track` element with a `kind="descriptions"` attribute.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video contains audio and a `track` element with a `kind="descriptions"` attribute.
```
Comment (nitedog) [inline _rules/SC1-2-Video-description-track.md]: same as above - question on any vs every and grammar issue

File: _rules/SC1-2-video-audio-description.md [hunk 27]
```diff
@@ -18,39 +19,39 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video contains audio.
```
Comment (nitedog) [inline _rules/SC1-2-video-audio-description.md]: same as above - question on any vs every and grammar issue

File: _rules/SC1-2-video-media-alternative.md [hunk 28]
```diff
@@ -0,0 +1,150 @@
...
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element that is [visible](#visible), where the video  contains audio.
```
Comment (nitedog) [inline _rules/SC1-2-video-media-alternative.md]: Grammar is correct here. Still the question about any vs every.

File: _rules/SC1-2-video-transcript.md [hunk 29]
```diff
@@ -21,45 +22,40 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible) where the video contains audio.
```
Comment (nitedog) [inline _rules/SC1-2-video-transcript.md]: same as above - question on any vs every and grammar issue

File: _rules/SC1-2-3-video-alternative.md (hunk 24 above)
Comment (WilcoFiers) [reply _rules/SC1-2-3-video-alternative.md]: Good point, changing this to "every".

File: _rules/SC1-2-video-audio-description.md (hunk 22 above)
Comment (WilcoFiers) [reply _rules/SC1-2-video-audio-description.md]: Good point. Fixing this now.

File: _rules/SC1-2-3-video-alternative.md [hunk 30]
```diff
@@ -0,0 +1,183 @@
...
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to every [non-streaming](#non-streaming) `video` element that is [visible](#visible), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
```
Comment (nitedog) [inline _rules/SC1-2-3-video-alternative.md]: After discussion with @WilcoFiers, this condition needs to change to something like:

> Each test target, needs to meet the expectations defined by at least one of the following rules:

This applies to *all* composite rules going forward, so not sure how to handle it here (leave for now until this is discussed with the group vs make the change now already for others to follow).

File: _rules/SC1-2-1-audio-transcript.md [hunk 31]
```diff
@@ -20,7 +20,7 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is:
+The rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 - playing; or,
 - has a "play button" that is [visibile](#visible) and [included-in-the-accessibility-tree](#included-in-the-accessibility-tree).
```
Comment (nitedog) [inline _rules/SC1-2-1-audio-transcript.md]: For the expectation below (sorry for not catching this earlier), I suggest rewording to:

> The auditory information of each test target is available through a text transcript. That text transcript is [visibile](#visible) and [included-in-the-accessibility-tree](#included-in-the-accessibility-tree), either on the page or through a link.

Main change is to remove the "needs to" because that does not match the wording for an expectation. Also minor editorial re-arrangement for readability.

File: _rules/SC1-2-3-video-alternative.md (hunk 30 above)
Comment (WilcoFiers) [reply _rules/SC1-2-3-video-alternative.md]: Auto-WCAG rules are written according to the latest published draft of the ACT Rules Format. What you are referring to is a discussion that hasn't been resolved yet, let alone published. We can't expect rule writers to keep up with internal ACT discussions, even if those are on public mailing lists.

We'll update rules when there is a new public draft / CR.

**Comment by WilcoFiers** [comment]
@nitedog I think I've addressed your questions, can you have another look?

File: _rules/SC1-2-3-video-alternative.md (hunk 30 above)
Comment (nitedog) [reply _rules/SC1-2-3-video-alternative.md]: OK
